.gitignore

```diff
@@ -6,6 +6,7 @@
 .f2py_f2cmap
 .idea/
 .vscode/
+src/qutip_cupy/*.cpp
```

Comment:
I usually add project specific ignores like this one right at the end of the `.gitignore` file (so that they're easy to find and clean up later if needed).

Reply:
you can also add it into the `.gitignore` that's in `src/qutip_cupy`.  If you don't like the multiple `.gitignore`, then you should delete the one in` src/qutip_cupy` to avoid fracturing the concerns.

Reply:
Good points. I added it in the `.gitignore`  inside  `src/qutip_cupy` at the end of the file. Looked like the right place.